### ISSUE TITLE
Sliding interval and fade effect doesn't work correctly

### ISSUE THREAD

**Issue by bennie99** [fritzmg/contao-swiper#6]
I'm using this settings in Content slider (Wrapper start):
Sliding interval :1000
and  Effect: Fade
Both does not work,
The Slide Effect does not start automatically as expected (it did in Vers 0.4.1) and the Fade Effect isn't really like that, it is more like a Overlay (one Pic over the other), maybe there is css missing?
Maybe I did something wrong but what?

**Comment by tblumrich** [comment]
Hallo!
Ich kann das Problem exakt bestätigen. Egal was ich bei Slide-Intervall ausser 0 eintrage, er slidet nicht automatisch... und ja, beim Fade stapeln sich alle Grafiken anfangs übereinander.
Ich hab das Modul schon in anderen Projekten in Benutzung, ohne Probleme dieser Art. (Version 1.0.0 z.B.)
Wäre schön, wenn du das prüfen könntest. Kann auch gern mein derzeitiges Textprojekt zur Verfügung stellen.

**Comment by fritzmg** [comment]
I'll have a look at it. In the mean time just use `^1.0`, since `^2.0` does not offer that much new features.

**Comment by Xirdion** [comment]
The autoplay-problem in v2.0 should be fixed with adding something like this to the `contao-swiper.js` file according to the swiper docs. (http://idangero.us/swiper/api/#autoplay)
```
        if (params.autoplay) {
            params.autoplay = {
                delay: params.autoplay,
            }
        }
```

**Comment by fritzmg** [comment]
I have just released [`3.0.0`](https://github.com/fritzmg/contao-swiper/releases/tag/3.0.0), which introduces a lot more changes. If you had custom templates before, you will need to update those templates.

In the new release both of these problems should not exist anymore.
